release/community-stable/archlinux/docker/Dockerfile

```diff
@@ -10,6 +10,45 @@ ARG imageRepo=archlinux/base
 
 FROM ${imageRepo}:${fromTag} AS installer-env
 
+RUN \
+    # create a builduser
+    useradd builduser -m \
+    # delete the builduser password
+    && passwd -d builduser \
+    # enable builduser to sudo without password
+    && printf 'root ALL=(ALL) ALL\n' | tee -a /etc/sudoers \
+    # update package database
+    && pacman -Syy \
+    # install dependencies
+    && pacman -S --noconfirm \
+      # required sudo
+      sudo \
```

Comment:
That's interesting, didn't know this was an actual package - thought it was built in haha

Reply:
Apparently, `sudo` is missing if you use 'archlinux/base:latest' container image.

```sh
[root@42b7c3b2ff4c /]# sudo
bash: sudo: command not found
```

And yes, `sudo` command is originally from [sudo-1.8.28-1](https://www.archlinux.org/packages/core/x86_64/sudo/) package for ArchLinux.

Haha... Just like a certain percentage of the world population believes that IE is builtin in Windows.